gen3/bin/reset.sh

```diff
@@ -145,8 +170,10 @@ g3kubectl create configmap fence "--from-file=user.yaml=$useryaml"
 # try to make reset more reliable - especially in Jenkins
 #
 run_setup_jobs
-remove_wts_creds_secrets
+clear_wts_clientId
 gen3 roll all
+reset_wts_clientId
```

Comment:
there is no `reset_wts_clientId`, do you mean `new_wts_clientId` ?

Reply:
doh!